shared-libs/transitions/src/transitions/registration.js

```diff
@@ -40,6 +41,22 @@ const getNameField = (params, prefix) => {
   return defaultNameField;
 };
 
+const getPatientPhoneField = (currentForm) => {
+  const formDef = utils.getForm(currentForm);
+  if (!formDef) {
+    return;
+  }
+
+  if (formDef && formDef.fields) {
```

Comment:
We can further simplify this like: 
```suggestion
if (!formDef?.fields) {
  return;
}

return Object
  .keys(formDef.fields)
  .find(key => formDef.fields[key].type === 'phone_number');

```

Reply:
Thanks. 
P.S. eslintrc for shared-libs was in 2018 which coudln't recognize the optional chaining syntax (?.) so upgraded it to 2020. 3 or 4 folders are already in 2020.